firefox-ios/Client/Frontend/Browser/EnhancedTrackingProtection/EnhancedTrackingProtectionVC.swift

```diff
@@ -72,11 +72,7 @@ class EnhancedTrackingProtectionMenuVC: UIViewController, Themeable {
     }
 
     private let siteDomainLabel: UILabel = .build { label in
-        label.font = DefaultDynamicFontHelper.preferredFont(
-            withTextStyle: .largeTitle,
-            size: ETPMenuUX.Fonts.websiteTitleFontSize,
-            weight: .semibold
-        )
+        label.font = FXFontStyles.Regular.largeTitle.scaledFont()
```

Comment:
@cwzilla for this view, we are using `largeTitle` textStyle, but our font sizes we're using `15` and `10`. Can you confirm what we should be using while following the design system as well?

Reply:
Is it possible to see what this would look like with the body style?

Reply:
yes sure, here domainLabel is with body style


<img width="361" alt="image" src="https://github.com/mozilla-mobile/firefox-ios/assets/59967463/2a00e890-3e15-4b9c-8a60-396af975b1a4">

Reply:
Thank you so much @rajparad! Is this PR just for the domain label or should I be looking at the other elements/text on this screen too?

Reply:
No this PR is for full page. It's just to give you an idea how domain label looks, compare to other label with **body** style text.
 I am also attaching image below with full page in **body** style.
<img width="373" alt="image" src="https://github.com/mozilla-mobile/firefox-ios/assets/59967463/df44032e-cf3a-4d92-b7ad-68ec53a7b210">

Reply:
I can update all elements with `body` style if above image is fine!

Reply:
Thank you for sharing the changes! Everything looks good with body – just one more change: for "Protections are ON for this site," could we change this to `caption1`?

Reply:
Sure, looks good with `caption1` style. I have updated code, and the screenshots are below:

![image](https://github.com/mozilla-mobile/firefox-ios/assets/59967463/abe53c32-916d-4884-8e79-53c2cd8cadb4)

![image](https://github.com/mozilla-mobile/firefox-ios/assets/59967463/d88697ec-0de4-4fdb-8b21-a2ad96d4e418)